autojustice.lua

```diff
@@ -0,0 +1,297 @@
+-- Auto Justice
+--@enable = true
```

Comment:
you'll also need `@module = true`, otherwise the `isEnabled()` function can't be called and this script won't show up as enableable.